### ISSUE TITLE
Limit photos and videos permission

### ISSUE THREAD

**Issue by niporasomo** [thunder-app/thunder#348]
Is your feature request related to a problem? Please describe.

Please correct me if I'm wrong, but it seems like the app requests the permission for access to all photos and videos on a device? I think it would be a nice enhancement to the privacy of the app if it would only have/request access to its own folder for storing downloads.

Describe the solution you'd like

If possible, have Thunder simply save media to its own folder without needing to request permissions for all photos and videos on the device. I think this is possible with scoped storage on Android? I believe there are other open source apps like Infinity for Reddit and Stealth that are able to do this, and are also on F-droid. Again, please forgive my ignorance on the topic as I'm not a developer and I could be mistaken.

Thanks again for making the app!

**Comment by hjiangsu** [comment]
That might be possible! Maybe someone who's more knowledgeable on Android permissions can pitch in?

For iOS devices, this is already implemented, but it would be good to see the same happen on Android

**Comment by niporasomo** [comment]
Yeah it would be nice to have! Perhaps someone else could help with it, I wasn't sure if the code for the other two apps would be worth looking at to see maybe if there were certain things they implemented.

**Comment by CTalvio** [comment]
Won't Thunder need read access as well, in order to upload media for posts?

On a related note, once we have that working, it would ve nice to have Thunder appear in the share options in other apps.

Would make posting articles from browsers and media from galleries a lot smoother.
